siatest/localfile.go

```diff
@@ -24,8 +24,8 @@ type (
 // NewFile creates and returns a new LocalFile. It will write size random bytes
 // to the file and give the file a random name.
 func NewFile(size int) (*LocalFile, error) {
```

Comment:
ah, looks like the `NewFile` functions aren't methods of any particular object. I wonder if that doesn't limit the ability to tell which object / test made which file?

Reply:
Yeah, as I mentioned above, they all land in the same folder since you can't really tell. Passing a path seems to defeat the simplicity this method tries to provide by abstracting everything except for the filesize. Maybe we can find a more satisfying solution than this.